content/docs/command-reference/remote/modify.md

```diff
@@ -857,69 +857,70 @@ Read more about by expanding the WebHDFS section in
 by HDFS. Read more about by expanding the WebHDFS section in
 [`dvc remote add`](/doc/command-reference/remote/add#supported-storage-types).
 
-- `url` - remote location:
+- `url` - remote location.
```

Comment:
?

Reply:
Nitpick on my end but I feel that either it should end in a period, or `example:`, just a floating colon on its own at the end looks weird. But again, not very attached to this change.